myft-digest-promo/main.scss

```diff
@@ -1,108 +1,115 @@
-$icon-width-s: 90px;
-$icon-width-m: 90px;
+@if nUiApply('myftDigestPromo') {
 
-.n-myft-digest-promo {
-	@include clearfix();
+@include nUiCritical('myftDigestPromo') {
+		.n-myft-digest-promo {
+			display: none;
+		}
+};
 
-	display: none;
-	float: none;
-	position: relative;
-	background-color: getColor('warm-5');
-	border-bottom: 1px solid getColor('warm-3');
-	border-top: 1px solid getColor('warm-3');
+	$icon-width-s: 90px;
+	$icon-width-m: 90px;
 
-	&.n-myft-digest-promo--enabled {
-		display: block;
-	}
+	.n-myft-digest-promo {
+		@include clearfix();
 
-	.n-myft-digest-promo__container {
+		float: none;
 		position: relative;
-		@include oGridRespondTo('M') {
-			@include oGridCenter;
-			@include oGridOffset(2);
+		background-color: getColor('warm-5');
+		border-bottom: 1px solid getColor('warm-3');
+		border-top: 1px solid getColor('warm-3');
+
+		&.n-myft-digest-promo--enabled {
```

Comment:
We shouldn’t need to chain the classes here; should be ok to have the `.n-myft-digest-promo--enabled` as its own block, outside and under this `.n-myft-digest-promo` block